checkout/application/orderService.go

```diff
@@ -62,11 +62,59 @@ const (
 	LastPlacedOrderSessionKey = "orderservice_last_placed"
 )
 
-var orderFailedStat = stats.Int64("flamingo-commerce/orderfailed", "my stat records 1 occurrence per error", stats.UnitDimensionless)
+var (
+	// orderValidationFailCount counts validation failures on orders
+	orderValidationFailCount = stats.Int64("flamingo-commerce/checkout/order_validation_failed", "Count of failures while validating orders", stats.UnitDimensionless)
```

Comment:
maybe rename since it's not an invalid order but an invalid cart

Reply:
Ok, I renamed it.